setup.py

```diff
@@ -46,12 +58,14 @@ def read(filepath):
     ],
     extras_require=dict(
         test=[
+            'Webtest',
```

Comment:
Where is webtest used?

Reply:
A good question, in `zope.app.wsgi.testlayer`. `zope.app.wsgi[testlayer]` solves this better.